generator/.DevConfigs/9903b621-ba47-4a5a-92ec-1bfbfadb70f7.json

```diff
@@ -0,0 +1,11 @@
+{
+    "services": [
+        {
+            "serviceName": "DynamoDBv2",
+            "type": "minor",
+            "changeLogMessages": [
+                "Added the ability to create observability traces during DynamoDB operations"
```

Comment:
To make sure it is clear this is for the HLL how about something like
```
Added support for creating observability traces from the DataModel and DocumentModel abstractions for DynamoDB
```

Reply:
Done